solutions/3728/01.py

```
@@ -0,0 +1,37 @@
+class Solution:
+    def countStableSubarrays(self, capacity: List[int]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)